fault/system_verilog_target.py

```diff
@@ -681,6 +703,9 @@ def generate_code(self, actions, power_args):
             port_list.extend(result)
         port_list = f',\n{2*self.TAB}'.join(port_list)
 
+        if len(self.pysv_funcs) > 0:
+            self.imports.append("pysv")
```

Comment:
Looking at 538, which guards against adding pysv more than once, could this add a pysv import more than once? Perhaps we could just simply make `imports` a set?

Reply:
I was worried about non-deterministic code generation result. It seems like none of the tests are directly comparing the code output so I think using set would be better.